src/Compilers/CSharp/Portable/CSharpResources.Designer.cs

```diff
@@ -5831,6 +5831,15 @@ internal class CSharpResources {
         }
         
         /// <summary>
+        ///   Looks up a localized string similar to &apos;{0}&apos; cannot implement interface member &apos;{1}&apos; in type &apos;{2}&apos; because feature &apos;{3}&apos; is not available in C# {4}. Please use language version {5} or greater..
+        /// </summary>
+        internal static string ERR_LanguageVersionDoesNotSupportDefaultInterfaceImplementationForMember {
```

Comment:
>ERR_LanguageVersionDoesNotSupportDefaultInterfaceImplementationForMember [](start = 31, length = 72)

Please remember to add error code ERR_LanguageVersionDoesNotSupportDefaultInterfaceImplementationForMember to CSharpUpgradeProjectCodeFixer and test. We should do that for all diagnostics that say "please use langver x or higher".
Separate PR is fine.

See [https://github.com/dotnet/roslyn/pull/18045/files#diff-d41456740132e589ef099473fd0d50ab](https://github.com/dotnet/roslyn/pull/18045/files#diff-d41456740132e589ef099473fd0d50ab)